src/cloud/services/RecyclingCenterService.ts

```diff
@@ -56,7 +56,7 @@ const editRecyclingCenter = async (
 
   if (role) {
     await role.fetch();
-    if (role.get('name') !== name) {
+    if (role.get('name') !== nameNormalize) {
```

Comment:
normalizedName?

Reply:
how we store the name of the role for that factory, e.g.: Itambe Mini -> itambe_mini

Reply:
change the variable name